docs/build-new/plot-quantum-states.ipynb

```diff
@@ -0,0 +1,508 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "fe606dcf-9205-4eb6-a789-0d66ae7dddec",
+   "metadata": {},
+   "source": [
+    "# Plot quantum states"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e0242a86-db2b-4cdd-835e-4aa8a5349fb8",
+   "metadata": {},
+   "source": [
+    "In many situations – such as learning or debugging – it's helpful to visualize the state of a quantum computer. Here we assume you already have a particular state from simulation or state tomography. These visualizations require resources exponential in the number of qubits, so it's only possible to view the states of small quantum systems.\n",
```

Comment:
Feels a bit awkward - possible to reword?

> These visualizations require resources exponential in the number of qubits,

Reply:
Could leave that bit out and just say

> It's only possible to view the states of small quantum systems.

?

Reply:
That works! 👍